components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -132,6 +143,11 @@
                             return false;
 
                         }
+                        if (!isWhiteListed(backChannelLogoutUrl, ["url"]) || !isNotBlackListed(backChannelLogoutUrl,
```

Comment:
why can't we have a method like isBlackListed() so we can avoid Not of an isNotBlackListed()